echopype/convert/parse_azfp.py

```diff
@@ -129,10 +129,10 @@ def load_AZFP_xml(self):
 
         def get_value_by_tag_name(tag_name, element=0):
             """Returns the value in an XML tag given the tag name and the number of occurrences."""
-            return px.getElementsByTagName(tag_name)[element].childNodes[0].data
+            return root.iter(tag_name).__next__().text
 
-        xmlmap = fsspec.get_mapper(self.xml_path, **self.storage_options)
-        px = xml.dom.minidom.parse(xmlmap.fs.open(xmlmap.root))
+        et = ET.parse(self.xml_path)
```

Comment:
Hi @praneethratna! Thanks for this much cleaner implementation. I do have a question, does `ET.parse` allow for remote files? so if my xml is located at `http://mysite.com/myxml.xml` would this read and load that xml into an Element Tree object?

Reply:
@lsetiawan I don't think so, i missed this point - we have to use `fsspec` and pass `xmlmap.fs.open(xmlmap.root)` as input to `ET.parse`.

Reply:
Yeah, it looks like you should retain the `xmlmap` statement (line 34), then, as you said, pass `xmlmap.fs.open(xmlmap.root)` as input to `ET.parse`. You can probably use a single, longer statement:
```python
root = ET.parse(xmlmap.fs.open(xmlmap.root)).getroot()
```